### ISSUE TITLE
Warning: OptionParser::InvalidOption: invalid option: --sourcemap

### ISSUE THREAD

**Issue by liampmccabe** [yeoman/generator-webapp_DEPRECATED#444]
Fresh install with Sass and Modernizer. After running 'grunt serve':

```
    Warning: OptionParser::InvalidOption: invalid option: --sourcemap
  Use --trace for backtrace. Use --force to continue.

    Aborted due to warnings.
```

**Comment by silvenon** [comment]
Which version are you using? 0.5.0?

**Comment by liampmccabe** [comment]
Yup.

**Comment by silvenon** [comment]
What happens when you remove `sourcemap: true` from the `sass` task?

**Comment by tmarrs** [comment]
That works properly.

**Comment by liampmccabe** [comment]
@silvenon Seems to fix the issue :)

**Comment by silvenon** [comment]
Yay!

**Comment by tmarrs** [comment]
I do not consider this issue Closed because this is a work-around, and not a real fix to the problem. 

This generator does not work out-of-the-box. After creating the project with this generator, I should just be able to run "grunt serve" and it should just work automatically without having to figure it out and fix Gruntfile.js manually.

Please re-open this issue and fix Gruntfile.js to remove "sourcemap: true" from the sass task.

**Comment by silvenon** [comment]
This was already fixed in 8e72360456acbf5e3720b991b9c82b5c7c9be877 (not yet released), that's why I closed the issue.
